open-craft/opencraft
# Remove swampdragon

This PR strips out all swampdragon and the functionality that it provided.

**Testing instructions**:

1. set up environment ready to create instances
2. make sure there is at least one instance reference to work with
3. select an instance and click "Launch new AppServer"
4. wait, refresh page, observe that new appserver has been created
5. check "Log" tab to verify that logs load
6. if appserver created successfully, select in and expand Log accordian. Verify that appserver logs load.
7. refresh page and expand Log accordian again. Verify that new log entries appear since last load.
